### PR TITLE
Get hw feature

### PR DESCRIPTION
**Pull request by PeiYee88**
Implement get hospital wing feature

Current commands within get:
get /hw for hospital wing
get /nn for name

### CODE REVIEW

**Review by nehcuy — APPROVED** [review]
LGTM!

File: src/main/java/seedu/address/logic/commands/GetHospitalWingCommand.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+    private final HospitalWingContainsKeywordsPredicate predicate;
+
+    public GetHospitalWingCommand(HospitalWingContainsKeywordsPredicate predicate) {
+        this.predicate = predicate;
+    }
+
+    @Override
+    public CommandResult execute(Model model) {
+        requireNonNull(model);
```
Comment (kxrt) [inline src/main/java/seedu/address/logic/commands/GetHospitalWingCommand.java:30]: I think in the future assertions might be good to add near certain requireNonNull(...) methods too!

File: src/main/java/seedu/address/logic/parser/AddressBookParser.java [hunk 2]
```diff
@@ -26,6 +29,8 @@ public class AddressBookParser {
      * Used for initial separation of command word and args.
      */
     private static final Pattern BASIC_COMMAND_FORMAT = Pattern.compile("(?<commandWord>\\S+)(?<arguments>.*)");
+    private static final Pattern GET_COMMAND_FORMAT = Pattern
```
Comment (kxrt) [inline src/main/java/seedu/address/logic/parser/AddressBookParser.java:32]: Nice regex 👍

File: src/main/java/seedu/address/logic/parser/CliSyntax.java [hunk 3]
```diff
@@ -7,11 +7,11 @@ public class CliSyntax {
 
     /* Prefix definitions */
     public static final Prefix PREFIX_NAME = new Prefix("n/");
+    public static final Prefix PREFIX_NAME_PARSER = new Prefix("/nn");
```
Comment (kxrt) [inline src/main/java/seedu/address/logic/parser/CliSyntax.java:10]: Good job on following the parser prefix syntax

**Review by kxrt — APPROVED** [review]
Looks ready to merge!

**Pull request merged**